OpenTabletDriver.Configurations/Configurations/Huion/new 1060 plus 2048.json

```
@@ -0,0 +1,43 @@
+{
+  "Name": "Huion new 1060 plus (2048)",
```

```suggestion
  "Name": "Huion New 1060 Plus (2048)",
```
and rename the file to `New 1060 Plus (2048)`.